crates/astria-sequencer-utils/src/genesis_parser.rs

```diff
@@ -58,23 +62,19 @@ impl GenesisParser {
 
 /// Merges a source JSON Value into a destination JSON Value context:
 // https://stackoverflow.com/questions/47070876/how-can-i-merge-two-json-objects-with-rust
```

Comment:
this comment probably does not make sense anymore

Reply:
removed